extensions/aproc/proc/ingest/drivers/impl/cosmoskymed.py

```diff
@@ -48,34 +51,34 @@ def identify_assets(self, url: str) -> list[Asset]:
             self.thumbnail_path = thumbnail_path + '/thumbnail.jpg'
             geotiff_to_jpg(self.browse_path, 50, 50, self.thumbnail_path)
             assets.append(Asset(href=self.thumbnail_path,
-                                roles=[Role.thumbnail.value], name=Role.thumbnail.value, type="image/jpg",
+                                roles=[Role.thumbnail.value], name=Role.thumbnail.value, type=MimeType.JPG.value,
                                 description=Role.thumbnail.value, size=get_file_size(self.thumbnail_path), asset_format=AssetFormat.jpg.value))
             quicklook_path = self.output_folder + '/' + self.get_item_id(url) + '/quicklook'
             os.makedirs(quicklook_path, exist_ok=True)
             self.quicklook_path = quicklook_path + '/quicklook.jpg'
             geotiff_to_jpg(self.browse_path, 250, 250, self.quicklook_path)
             assets.append(Asset(href=self.quicklook_path,
-                                roles=[Role.overview.value], name=Role.overview.value, type="image/jpg",
+                                roles=[Role.overview.value], name=Role.overview.value, type=MimeType.TIFF.value,
```

Comment:
`image/jpg` => `MimeType.TIFF.value`

not the same type after refactor

Reply:
well spotted, thank you!